web/docs/Backend Starters/Decoupled Drupal/creating-new-project.md

```diff
@@ -1,25 +1,162 @@
 ---
+id: "creating-new-drupal-project"
+title: "Creating a New Drupal Project"
+slug: "/Backend Starters/Decoupled Drupal/Creating a New Project"
 sidebar_position: 2
 ---
 
-# Creating a New Project
+## Choosing an Approach
 
-## Prerequisites
+### Use Build Tools if:
+
+- Testing is an important part of your workflow
+
+- You don’t want to worry about manually pushing changes to your GH repo.
+
+### Use Dashboard Upstream if:
+
+- The simplest possible setup
+
+- Pantheon repo is your source of truth
+
+- GitHub is your default VCS and CircleCI is your CI provider then this approach will automatically default to both these options. For other options use the build tools approach
+
+### Installing using Dashboard Upstream
+- Create an empty upstream site:
+
+#### Via the Pantheon Dashboard at this link: 
+- https://dashboard.pantheon.io/sites/create?upstream_id=4c7176de-e079-eed1-154d-44d5a9945b65
+
+#### Or Alternatively via Terminus:
+
+```
+terminus site:create my-new-site "Describe Site" empty --org='My Team Name'
+```
+
+Replace `'{My Team Name}'` with your team name - for example `My Agency`. This can also be omitted.
+
+#### In the Pantheon Admin Dashboard:
+
+Switch **Development Mode** to **Git**
+
+Click the **Upgrade** button, and change the site plan to **Performance** > **Medium**. 
+
+** Clone your sites repo on your local
+
+** Change into the newly cloned directory
+
+** Add remote for the decoupled upstream: `git remote add upstream https://github.com/pantheon-systems/decoupled-drupal-composer-managed`
+
+** Checkout the main branch from upstream:
+
+```
+
+git fetch upstream main
+
+git checkout -t upstream/main
+
+```
+
+** Reset the master branch to match main, then force push that to the Dev environment:
+
+```
+
+git branch -D master
+
+git checkout -b master
+
+git push origin master --force
+
+```
+
+:::Note
+Make sure code has synced and all active workflows are complete in the UI.
+Click on the **Visit Development Site** button to Install via the UI—selecting either the Pantheon Decoupled Profile, or Pantheon Decoupled Umami Demo profiles.
+:::
+
+## Installing using Build Tools
+### Prerequisites
 
 - Composer (required for CMS backends): [Install Globally](https://getcomposer.org/download/)
 - [Generate machine token](https://pantheon.io/docs/machine-tokens#create-a-machine-token) & [Authenticate into Terminus](https://pantheon.io/docs/machine-tokens#authenticate-into-terminus)
 - [Install Terminus](https://pantheon.io/docs/terminus/install) (3.0.0 above required)
 - Also install the following plugins:
   - `terminus self:plugin:install terminus-build-tools-plugin`
-  - `terminus self:plugin:install terminus-power-tools`
+  - `terminus self:plugin:remove terminus-power-tools`
```

Comment:
I think this case will be very rare. Maybe it would be best to include this in a 'known issues' section at the end.